monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md

```diff
@@ -0,0 +1,129 @@
+# MONERO
+
+Monero est une cryptomonnaie sécurisée,
+confidentielle et intraçable disponible pour
+tous
+
+# QUELQUES FAITS
+
+Les utilisateurs peuvent la dépenser en
+toute sécurité, personne ne peut voir leurs
+soldes ou suivre leurs activités.
+
+Révisé le 14/11/2018 - [Version Imprimable](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)
+Créé pour la communauté par Monero Outreach
+
+[getmonero.org](https://getmonero.org/)
```

Comment:
This is not incorrect but we could fix it to add the correct link for each language, in this case it would be:

French getmonero.org homepage
https://www.getmonero.org/fr/index.html